### PR TITLE
Companion for: `try-runtime::fast-forward`

### PR DESCRIPTION
**Pull request by pmikolajczyk41**
companion for https://github.com/paritytech/substrate/pull/12896

### CODE REVIEW

File: cli/src/command.rs [hunk 1]
```diff
@@ -27,6 +27,9 @@ use sp_core::crypto::Ss58AddressFormatRegistry;
 use sp_keyring::Sr25519Keyring;
 use std::net::ToSocketAddrs;
 
+#[cfg(feature = "try-runtime")]
+use try_runtime_cli::block_building_info::timestamp_with_babe_info;
```
Comment (kianenigma) [inline cli/src/command.rs:31]: I would move this to line 641 inside the feature-gated scope. There is already a few imports there.

**Review by kianenigma — APPROVED** [review]

File: cli/src/command.rs (hunk 1 above)
Comment (pmikolajczyk41) [reply cli/src/command.rs]: sure, moved

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2343379

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2369942

**Review by sam0x17 — APPROVED** [review]

**Review by Ank4n — APPROVED** [review]

File: cli/src/command.rs [hunk 2]
```diff
@@ -657,18 +660,20 @@ pub fn run() -> Result<()> {
 				<E as NativeExecutionDispatch>::ExtendHostFunctions,
 			>;
 
-			use sc_service::TaskManager;
 			let registry = &runner.config().prometheus_config.as_ref().map(|cfg| &cfg.registry);
 			let task_manager = TaskManager::new(runner.config().tokio_handle.clone(), *registry)
 				.map_err(|e| Error::SubstrateService(sc_service::Error::Prometheus(e)))?;
 
+			let info_provider = timestamp_with_babe_info(6000);
```
Comment (Ank4n) [inline cli/src/command.rs:667]: Would be nicer to not hardcode but get the block time value from each runtime itself.

File: cli/src/command.rs [hunk 3]
```diff
@@ -657,18 +660,20 @@ pub fn run() -> Result<()> {
...
+
 			ensure_dev(chain_spec).map_err(Error::Other)?;
 
 			#[cfg(feature = "kusama-native")]
 			if chain_spec.is_kusama() {
 				return runner.async_run(|_| {
 					Ok((
-						cmd.run::<service::kusama_runtime::Block, HostFunctionsOf<service::KusamaExecutorDispatch>>(
+						cmd.run::<service::kusama_runtime::Block, HostFunctionsOf<service::KusamaExecutorDispatch>, _>(
+							Some(info_provider)
```
Comment (Ank4n) [inline cli/src/command.rs:676]:
```suggestion
						cmd.run::<service::kusama_runtime::Block, HostFunctionsOf<service::KusamaExecutorDispatch>, _>(
							Some(timestamp_with_babe_info(service::kusama_runtime::MILLISECS_PER_BLOCK))
```
and similarly others.

Comment (pmikolajczyk41) [reply cli/src/command.rs]: great! thank you for this suggestion - first time I didn't realize that I have access to these parameters; the suggestion is now applied

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2381054

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2397073

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2406490

**Comment by kianenigma** [comment]
bot rebase

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2417245

**Pull request closed without merging**